### PR TITLE
KDE.kstars version 3.5.4

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25668)

### CODE REVIEW

File: manifests/k/KDE/kstars/3.5.4/KDE.kstars.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: KDE.kstars
+PackageVersion: 3.5.4
```
Comment (ImJoakim) [inline manifests/k/KDE/kstars/3.5.4/KDE.kstars.installer.yaml:5]: image

```suggestion
PackageVersion: stable-3.5.4-bf8eb4d81
```

File: manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KDE.kstars
+PackageVersion: 3.5.4
```
Comment (ImJoakim) [inline manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: stable-3.5.4-bf8eb4d81
```

File: manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KDE.kstars
+PackageVersion: 3.5.4
+PackageLocale: en-US
+Publisher: KDE
```
Comment (ImJoakim) [inline manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml:7]: image

```suggestion
Publisher: KDE e.V.
```

File: manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KDE.kstars
+PackageVersion: 3.5.4
+PackageLocale: en-US
+Publisher: KDE
+Author: KDE
+PackageName: kstars
```
Comment (ImJoakim) [inline manifests/k/KDE/kstars/3.5.4/KDE.kstars.locale.en-US.yaml:9]: image

```suggestion
PackageName: KStars Desktop Planetarium
```

File: manifests/k/KDE/kstars/3.5.4/KDE.kstars.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: KDE.kstars
+PackageVersion: 3.5.4
```
Comment (ImJoakim) [inline manifests/k/KDE/kstars/3.5.4/KDE.kstars.yaml:5]: image

```suggestion
PackageVersion: stable-3.5.4-bf8eb4d81
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
Please see my comments above. You also need to change the version folder:

`manifests/k/KDE/kstars/3.5.4/` -> `manifests/k/KDE/kstars/stable-3.5.4-bf8eb4d81/`

https://github.com/microsoft/winget-pkgs/pull/25668#discussion_r693488911
https://github.com/microsoft/winget-pkgs/pull/25668#discussion_r693488913
https://github.com/microsoft/winget-pkgs/pull/25668#discussion_r693488915
https://github.com/microsoft/winget-pkgs/pull/25668#discussion_r693488919
https://github.com/microsoft/winget-pkgs/pull/25668#discussion_r693488924

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**